src/components/View/Lottery/components/DrawInfo/Box.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import { Flex } from '@/components/Core/Toolkit'
+
+import { SvgSolidChevronDown } from '@/components/Svg'
```

Comment:
The way I sort the imports is as follows:
```suggestion
// The first group is nextjs and react internal packages.
import { useState } from 'react'

// The second group is external packages.
import styled from 'styled-components'

// And the third group is internal packages.
import { Flex } from '@/components/Core/Toolkit'
import { SvgSolidChevronDown } from '@/components/Svg'
```